webpack/assets/javascripts/react_app/redux/API/APIFixtures.js

```diff
@@ -26,6 +27,20 @@ export const action = {
   },
 };
 
+export const postActionWithCallback = {
+  type: API_OPERATIONS.POST,
+  payload: {
+    key: 'PREV_STATE',
```

Comment:
This key might be a bit confusing, how about `MY_RESOURCE`

Reply:
agree, though the reason for the `PREV` prefix is to emphasize  the **old** state (before fetching the data), I've changed it to ` INITIAL_RESOURCE`